openpype/hosts/nuke/api/plugin.py

```diff
@@ -804,8 +804,13 @@ def __init__(self,
         self.log.info("File info was set...")
 
         self.file = self.fhead + self.name + ".{}".format(self.ext)
-        if self.ext != VIDEO_EXTENSIONS:
-            self.file = os.path.basename(self.path_in)
+        if ".{}".format(self.ext) not in VIDEO_EXTENSIONS:
+            filename = os.path.basename(self.path_in)
+            self.file = filename
+            if ".{}".format(self.ext) not in self.file:
+                wrg_ext = filename.split(".")[-1]
```

Comment:
`wrg_ext` is a wrong extension or what exactly? If it is `wrong` than it is suggesting there is something wrong with it and It doesn't seem to be correct in this case. I would suggest to use `original_ext` also it is easier to read if it is full name rather than the abbreviation.